docs/source/torch.compiler.md

```diff
@@ -75,10 +77,11 @@ Some of the most commonly used backends include:
      - Uses Apache TVM for inference optimizations. `Read more <https://tvm.apache.org/>`__
    * - ``torch.compile(m, backend="openvino")``
      - Uses OpenVINO for inference optimizations. `Read more <https://docs.openvino.ai/torchcompile>`__
+```
 
-Read More
-~~~~~~~~~
+## Read More
 
+```{eval-rst}
```

Comment:
This file looks good but the list at the bottom is not showing as a list, no bullet points. @svekars is this a style issue?  

https://docs-preview.pytorch.org/pytorch/pytorch/155348/torch.compiler.html

Reply:
Do you mean the read more section? Seems same if comparing to main https://docs.pytorch.org/docs/main/torch.compiler.html ?